config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -1,27 +1,9 @@
 periodics:
-- name: periodic-cluster-api-provider-aws-test-creds
-  decorate: true
-  interval: 4h
-  labels:
-    preset-service-account: "true"
-    preset-aws-ssh: "true"
-    preset-aws-credential: "true"
-  extra_refs:
-  - org: kubernetes-sigs
-    repo: cluster-api-provider-aws
-    base_ref: master
-    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
-  spec:
-    containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200128-8dae294-master
-      command:
-      - "./scripts/ci-aws-cred-test.sh"
-  annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
-    testgrid-tab-name: test-creds
```

Comment:
Removes this test since it is testing against kops credentials and not the boskos-provided credentials we are using for other testing.